medallion/test/test_memory_backend.py

```diff
@@ -372,3 +376,190 @@ def test_saving_data_file(self):  # just for the memory backend
             )
             objs = self.load_json_response(r_get.data)
             assert objs["objects"][0]["id"] == new_id
+
+    def test_get_collections_401(self):
+        r = self.client.get("/trustgroup1/collections/")
+        self.assertEqual(r.status_code, 401)
+
+    def test_get_collections_403(self):
+        """not implemented
+
+        Medallion TAXII	implementation does not have access control for Collection resource metadata
+        """
+        pass
+
+    def test_get_collections_404(self):
+        r = self.client.get("/carbon1/collections/", headers=self.auth)
```

Comment:
Is this raising an error because of an unknown API root? If so, it might be good to put that in a comment or docstring.

Reply:
Yes. Will do.